.gitignore

```diff
@@ -26,3 +26,4 @@ testbin/*
 
 # Manually generated crds. Directory is needed though
 generated-crds/*
+vendor/
```

Comment:
why is this needed?

Reply:
Just to avoid adding the vendor directory in a commit.